### PR TITLE
94 guardian show list of guardians in federation

### PR DESCRIPTION
**Pull request by EthnTuttle**
Displays Peer Id and Name alongside connection status.
Fixes typo in API endpoint field name.

### CODE REVIEW

File: apps/guardian-ui/src/admin/FederationAdmin.tsx [hunk 1]
```diff
@@ -17,13 +17,48 @@ import {
...
 import { ReactComponent as CopyIcon } from '../assets/svgs/copy.svg';
 import { Pill } from '../components/Pill';
 
+interface PeerData {
+  id: number;
+  name: string;
+  status: PeerStatus;
+}
+
+function generatePeerData(
```
Comment (okjodom) [inline apps/guardian-ui/src/admin/FederationAdmin.tsx:41]: style: I'd just inline this function within the useEffect. It's not reused

File: apps/guardian-ui/src/admin/FederationAdmin.tsx [hunk 2]
```diff
@@ -17,13 +17,48 @@ import {
...
+  config: ClientConfig,
+  federationStatus: FederationStatus
+): PeerData[] {
+  const peerDataArray: PeerData[] = [];
+  for (const [id, status] of Object.entries(federationStatus.status_by_peer)) {
+    const numericId = parseInt(id, 10);
+    if (Object.prototype.hasOwnProperty.call(config.api_endpoints, numericId)) {
```
Comment (okjodom) [inline apps/guardian-ui/src/admin/FederationAdmin.tsx:48]: can we skip the outer `hasOwnProperty` check? It doesn't seem very useful if endpoint is still nullable and we can just go with 
```ts
const endpoint = config.api_endpoints[numericId];
      if (endpoint) {
        peerDataArray.push({
          id: numericId,
          name: endpoint.name,
          status,
        });
      }
      ```

**Comment by okjodom** [comment]
needs rebase after #144 ?

**Comment by EthnTuttle** [comment]
> needs rebase after #144 ?

I don't believe so since this works on a different component. #144 was working on the header status bar and this is the lower right card with Peer status. 
We should probably colorize the Peers in some way too?

**Review by wbobeirne — APPROVED** [review]
Worked great 👍 

Screenshot 2023-08-29 at 5 13 43 PM

**Comment by wbobeirne** [comment]
It did require a rebase though, I rebased and resolved conflicts.

**Pull request merged**